src/ngeo.js

```diff
@@ -249,3 +249,45 @@ ngeo.NumberType = {
    */
   INTEGER: 'integer'
 };
+
+/**
+ * @enum {string}
+ * @export
+ */
+ngeo.Permalink3dParam = {
```

Comment:
You should add the enum in the src/olcs directory.
I think this file should only contain general purpose objects. All what is specific to a particular use-case will be moved elsewhere, see https://jira.camptocamp.com/browse/GSGMF-223.